svnwiki-mode.el

```diff
@@ -1,11 +1,13 @@
-;;; svnwiki-mode.el --- Major mode for editing svnwiki markup
+;;; svnwiki-mode.el --- Major mode for editing svnwiki markup -*- lexical-binding: t -*-
 
 ;; Copyright (C) 2016 Vasilij Schneidermann <mail@vasilij.de>
+;; SPDX-License-Identifier: GPL-3.0-or-later
```

Comment:
Does MELPA actually require that? First time I've seen this in an Emacs package.

Reply:
Neither MELPA not GNU conventions require it, but there are [plans to use it](https://github.com/purcell/package-lint/issues?q=spdx) as a standard for MELPA, and we hope to convince emacs-devel as well. These identifiers are popular enough that they're now used in the Linux kernel, for example.